lib/smoke_detector/middleware/javascript_monitors.rb

```diff
@@ -31,7 +31,8 @@ def monitoring_code
     end
 
     def monitor?(headers)
-      headers["Content-Type"] =~ ACCEPTABLE_CONTENT && monitoring_code.present?
+      # minified code may not be UTF8 compliant, so you can't use present? reliably
+      headers["Content-Type"] =~ ACCEPTABLE_CONTENT && monitoring_code.size > 10
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.